fastlane/lanes/release.rb

```diff
@@ -88,14 +132,54 @@
   #####################################################################################
   desc 'Updates a release branch for a new beta release'
   lane :new_beta_release do |options|
-    android_betabuild_prechecks(base_version: options[:base_version], skip_confirm: options[:skip_confirm])
+    # Checkout default branch and update
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    # Check local repo status
+    ensure_git_status_clean
+
+    # Check versions
+    message = <<-MESSAGE
+
+      Current beta version: #{current_beta_version}
+      New beta version: #{next_beta_version}
+
+      Current build code: #{current_build_code}
+      New build code: #{next_build_code}
+
+    MESSAGE
+
+    # Check branch
+    unless !options[:base_version].nil? || Fastlane::Helper::GitHelper.checkout_and_pull(release: current_release_version)
+      UI.user_error!("Release branch for version #{current_release_version} doesn't exist.")
+    end
+
+    # Check user override
+    override_default_release_branch(options[:base_version]) unless options[:base_version].nil?
```

Comment:
I can't find this method in WPAndroid or `release-tooltkit` :( could you help me with that?

Reply:
This is now added at the end of `release.rb`. 

Addressed in https://github.com/wordpress-mobile/WordPress-Android/commit/6ab18efe9261a63b77766614c06c3bc661a697f6